pipelines/rights_classifier.py

```
@@ -0,0 +1,249 @@
+#!/usr/bin/env python3
+"""Mapping-authorized rights decisions and byte-bound envelope verification."""
+
+from __future__ import annotations
+
+import sys
+from dataclasses import dataclass
+from types import MappingProxyType
+from typing import Mapping
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)